packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActions.types.ts

```diff
@@ -0,0 +1,55 @@
+import { IButtonProps } from '../../Button';
+import { IStyle, ITheme } from '../../Styling';
+import { IRefObject, IStyleFunctionOrObject } from '../../Utilities';
+import { DocumentCardActionsBase } from './DocumentCardActions.base';
+
+export interface IDocumentCardActionsProps extends React.ClassAttributes<DocumentCardActionsBase> {
+  /**
+   * Gets the component ref.
+   */
+  componentRef?: IRefObject<{}>;
```

Comment:
nit: Not sure about it but better to have an empty interface of `IDocumentCardActions` and pass it to `IRefObject` in here. There are many components following this pattern. This way you have a type-safe hook in the future if you want to add some publicly accessible methods on the instance of the component.